docs/oss-oncall-log.md

```diff
@@ -0,0 +1,13 @@
+# OSS oncall log
+
+"Oh yeah, I remember having to deal with that a while back... what was it again?"
+
+- Issue: bigquery metrics dashboard in velodrome.k8s.io is out of date ([#8703](https://github.com/kubernetes/test-infra/issues/8703))
+  - Symptoms:
+    - velodrome.k8s.io is out of date
+    - go.k8s.io/triage is out of date
+    - k8s-gubernator BigQuery table is out of date
+  - Remediation:
+    - Kettle froze and needed to be restarted
+    - See [Kettle's README](/kettle/README.md)
```

Comment:
- added an alert on velodrome when no rows have been added to the DB in the last 6 hours http://velodrome.k8s.io/dashboard/db/bigquery-metrics?orgId=1&panelId=12&fullscreen

Reply:
I added a note to this firing under symptoms